core/dbt/events/functions.py

```diff
@@ -52,10 +44,6 @@
 
 
 def setup_event_logger(log_path, level_override=None):
-    # flags have been resolved, and log_path is known
```

Comment:
I've removed setting the event_history here because of discussions we have about events not lasting for multiple calls. Does this make sense? Would it be better to reset it here too?

Reply:
Yes! Makes sense that `EVENT_HISTORY` wants to last for the duration of the application, rather than just the duration of the invocation